tool_run_scripts/anvill.py

```diff
@@ -267,10 +267,10 @@ def anvill_python_main(args, source_path, dest_path):
 
     # get all the bitcode
     log.info(f"Listing files in {str(source_path)}")
-    sources = list(source_path.rglob("*.elf"))
-    # Sometimes we forget the .elf suffix
-    sources.extend(list(source_path.rglob("*.o")))
-    log.info(f"Found {len(sources)} ELF files")
+    # Filter for files that are executable
+    sources = [source for source in source_path.rglob("*") if os.path.isfile(source) and os.access(source, os.X_OK)]
```

Comment:
Is this a good idea, what about object files like in anghabench

Reply:
oh right, forgot about that
we could check both .o and executable files
or just make the .o files executable